specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -506,6 +649,203 @@
           }
         }
       }
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Active Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nGet all government-issued active storms.",
+        "operationId": "Weather_GetActiveStorms",
+        "x-ms-examples": {
+          "GetActiveStorms": {
+            "$ref": "./examples/GetActiveStorms.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/WeatherApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ActiveStormResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/{format}": {
+      "get": {
+        "description": "**Get government-issued Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nSearch government-issued storms by year, basin ID, and government ID.",
+        "operationId": "Weather_StormSearch",
+        "x-ms-examples": {
+          "StormSearch": {
+            "$ref": "./examples/GetStormSearch.json"
+          },
+          "StormSearch with three conditions": {
+            "$ref": "./examples/GetFullStormSearch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/WeatherApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ResponseFormat"
+          },
+          {
+            "$ref": "#/parameters/WeatherYear"
+          },
+          {
+            "$ref": "#/parameters/OptionalWeatherBasinID"
+          },
+          {
+            "$ref": "#/parameters/OptionalWeatherGovernmentID"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormSearchResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/forecasts/{format}": {
```

Comment:
not sure what we finally align on, forecasts or forecast?

Reply:
I think the conclusion is using forecasts, since we have agreed to use storms which is plural.